addons/binding/org.openhab.binding.modbus/src/main/java/org/openhab/binding/modbus/discovery/ModbusDiscoveryListener.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.modbus.discovery;
+
+import org.eclipse.smarthome.config.discovery.DiscoveryResult;
+
+/**
+ * Listener for discovery results
+ *
+ * Each discovered thing should be supplied to the thingDiscovered
+ * method.
+ *
+ * When the discovery process has been finished then the discoveryFinished
+ * method should be called.
+ *
+ * @author Nagy Attila Gabor - initial contribution
+ *
+ */
+public interface ModbusDiscoveryListener {
```

Comment:
[Class should have @NonNullByDefault](https://www.openhab.org/docs/developer/development/conventions.html#null-annotations). Mark @Nullable

Reply:
done